interpreter/cli/cli.py

```diff
@@ -8,6 +8,16 @@
 from ..utils.get_config import get_config_path
 from ..terminal_interface.conversation_navigator import conversation_navigator
 
+import sys
+import pysqlite3
+
+# Alias pysqlite3 as sqlite3 in sys.modules. this fixes a chromadb error where it whines about the wrong version being installed, but we cant change the containers sqlite.
+# 'pysqlite3' is a drop in replacement for default python sqlite3 lib. ( identical apis )
+sys.modules['sqlite3'] = pysqlite3
```

Comment:
Just putting a note here so that we don't forget to remove this when we switch to a different base container image.

Reply:
why can't the container sqlite be changed?